currency-converter/src/types.ts

```diff
@@ -0,0 +1,9 @@
+export interface IDropdownOption {
```

Comment:
these both are same. There's a minute difference in types and interfaces. We should just have one interface here, and use it in both components.